api/enterprise-api-ref/personal-groups.markdown

```diff
@@ -27,22 +27,17 @@ The personal groups API enables creating host groups based on host filters (the
     * **hostContextExclude** *(array)*  Optional parameter.
       Excludes results that concern hosts which have specified CFEngine context (class) set. Hosts that have at least one
       of the specified contexts set will be excluded from the results.
-  * **hostsFilter** *(json object)* Optional parameter.
-      * **includes** *(json object)* Optional parameter.
-        Object that specifies hosts to be included.
-          * **includeAdditionally** *(boolean)* Default: `false`
-            Defines if hosts will be included additionally on top of the inventory filters or class filters.
-          * **type** (string)
-            Now the only supported type is `hostkey`.
-          * **data** *(array)*
-            An array of hosts keys.
-      * **excludes** *(json object)* Optional parameter.
-        Object that specifies hosts to be excluded.
-          * **type** (string)
-            Now the only supported type is `hostkey`.
-          * **data** *(array)*
-            An array of hosts keys.
-
+    * **hostFilter** *(json object)* Optional parameter.
+        * **includes** *(json object)* Optional parameter.
+          Object that specifies hosts to be included.
+            * **includeAdditionally** *(boolean)* Default: `false`
+              Defines if hosts will be included additionally on top of the inventory filters or class filters.
```

Comment:
I don't immediately understand from this what happens in both true and false cases.

So includes with includeAdditionally=false won't include the hosts given by the current filter? What is the point of this flag?

Does false mean that the hosts found with the current filter will only be included if already part of inventory or class filters?

And so true means that the hosts found with the current filter will be included regardless of the results given by the inventory or class filters?

Reply:
maybe "on top" would be better said as 
```suggestion
              Defines if hosts will be added to the results returned by inventory filters or class filters.
```